pkg/subnet/local.go

```diff
@@ -172,6 +175,15 @@ func (d *SubnetDeployer) doDeploy(chain string, chain_genesis string) error {
 	fmt.Println()
 	firstURL := endpoints[0]
 
+	tokenName := "TEST"
+	sidecar, err := d.app.LoadSidecar(chain)
```

Comment:
Does this need to be applied to all `LoadSidecar` calls? If so, it might be nicer to just put this code inside `LoadSidecar`, to avoid every caller to have to do it.

Otherwise ignore this comment.